compiler/src/main/java/com/squareup/anvil/compiler/ModuleMergerIr.kt

```diff
@@ -53,15 +53,21 @@ internal class ModuleMergerIr(
             .findAll(mergeComponentFqName, mergeSubcomponentFqName, mergeModulesFqName)
             .ifEmpty { return super.visitClass(declaration) }
 
-          generateDaggerAnnotation(annotations, moduleFragment, pluginContext, declarationReference)
+          pluginContext.irBuiltIns.createIrBuilder(declaration.symbol)
+            .generateDaggerAnnotation(
```

Comment:
Creating an IR builder exposes more factory extension functions, including some of the ones used below

Reply:
Do we need to add any followup `build()` equivalent for the IrBuilder? Seems like the answer is no but just wanted to verify

Reply:
Not as far as I know, it's more of a context thing than a true builder